docs/resources/lb.md

```diff
@@ -81,27 +81,55 @@ resource "scaleway_instance_server" "main" {
 }
 
 ### IP for LB IP
-resource scaleway_lb_ip main {
+resource "scaleway_lb_ip" "main" {
 }
 
 ### Scaleway Private Network
-resource scaleway_vpc_private_network "main" {
+resource "scaleway_vpc_private_network" "main" {
     name = "private network with static config"
 }
 
-### Scaleway Load Balancer
-resource scaleway_lb main {
+resource "scaleway_vpc_private_network" "second" {
+  name = "private network with DHCP config"
+}
+```
+
+## Private Network with static config
+
+```terraform
+resource "scaleway_lb_ip" "main" {
+}
+
+resource "scaleway_vpc_private_network" "main" {
+    name = "MyTest"
+}
+
+resource "scaleway_lb" "main" {
     ip_id = scaleway_lb_ip.main.id
     name = "MyTest"
     type = "LB-S"
-
     private_network {
         private_network_id = scaleway_vpc_private_network.main.id
         static_config = ["172.16.0.100"]
     }
+}
+```
+
+### Scaleway Load Balancer with multiple private_network
+
+```terraform
+resource "scaleway_lb" "main" {
```

Comment:
Could you have self contained examples that can be copy and pasted in a stand alone way? You can see a preview of the document here: https://registry.terraform.io/tools/doc-preview